Library/Formula/creduce.rb

```diff
@@ -0,0 +1,93 @@
+class Creduce < Formula
+  homepage "http://embed.cs.utah.edu/creduce/"
+  url "http://embed.cs.utah.edu/creduce/creduce-2.2.1.tar.gz"
+  sha256 "c2fb1b00feafee20b467674679b68a58bc14556c2f373bb9688150f1fc740072"
+
+  head "https://github.com/rgov/creduce.git"
+
+  depends_on "astyle"
+  depends_on "delta"
+  depends_on "llvm" => "with-clang"
```

Comment:
this formula depends on llvm-config, which is not provided by the system AFAICT. bottling `creduce` would possibly remove the llvm dependency, if it were statically linked